src/Logary/Internals.fs

```diff
@@ -205,6 +221,43 @@ module internal Set =
     | (s : _ Set) when s.Count = 0 -> Some EmptySet
     | _ -> None
 
+type Timeout =
```

Comment:
I suppose there's no harm in this; I'm suggesting not making it explicit like this, but instead returning an Alt from all operations that can return this DU case and letting the caller worry about whether to time the operation out or not; and having the other side implement selective synchronous operations on its side, effectively making the timeout operation a way that two concurrent processes/jobs/alts can synchronise around something.